src/plugins/azgenerator/CodeModelAzImpl.ts

```diff
@@ -2323,4 +2348,29 @@ export class CodeModelCliImpl implements CodeModelAz {
         }
         return this._cliCoreLib;
     }
+
+    public get AzureCliFolder(): string {
+        return this.codeModel.language['az']?.['azureCliFolder']
+    }
+
+    public get azOutputFolder(): string {
+        return this.codeModel.language['az']?.['azOutputFolder']
+    }
+
+    public get IsCliCore() {
+        return this.codeModel.language['az']?.['isCliCore']? true: false;
+    }
+
+    public get SDK_NeedSDK() {
+        return this.codeModel.language['az']?.['sdkNeeded']? true: false;
+    }
+
+    public get SDK_IsTrack1() {
+        return this.codeModel.language['az']?.['sdkTrack1']? true: false;
+    }
+
+    public get SDK_NoFlatten() {
+        return this.codeModel.language['az']?.['sdkNoFlatten']? true: false;
+    }
```

Comment:
Why set it in codemodel? not just use a private properties of model? and use set/get function?

Reply:
Because the value was set in the merger.